docs/data/material/components/table/DataTable.tsx

```diff
@@ -40,8 +40,7 @@ export default function DataTable() {
       <DataGrid
         rows={rows}
         columns={columns}
-        pageSize={5}
-        rowsPerPageOptions={[5]}
+        paginationModel={{ page: 0, pageSize: 5 }}
```

Comment:
Are we sure about this change? Clicking on the page button no longer work:https://deploy-preview-36598--material-ui.netlify.app/material-ui/react-table/#data-table

<img width="203" alt="Screenshot 2023-04-30 at 20 16 36" src="https://user-images.githubusercontent.com/3165635/235396733-5aa757e6-5fed-493c-9006-2d89eac5675e.png">

I'm coming from: https://mui-org.slack.com/archives/C041SDSF32L/p1682708252800599 cc @cherniavskii for guidance.

Reply:
Right, I'll submit a PR 👍